frontend/packages/core/src/error.tsx

```diff
@@ -47,13 +50,36 @@ const ErrorText = styled(Typography)`
   font-size: 0.875rem;
 `;
 
-const ErrorPanel = styled(ExpansionPanel)`
+const Accordion = styled(MuiAccordion)`
   background-color: inherit;
   padding: 0px;
-  width: 100%;
 `;
 
-const CompressedError = ({ title, message }) => {
+export interface CompressedErrorProps {
+  title?: string;
+  message: string;
+}
+
+const findBreakpoint = (message: string): number => {
```

Comment:
do we want a unit test for this? or is the storybook enough here?

Reply:
I'll add these later :)